samples/CosmosDBIntegration/ReadThroughSamples/Models/CosmosDBListData.cs

```diff
@@ -0,0 +1,9 @@
+﻿ using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples.CosmosDB.Models
+{
+    public record CosmosDBListData(
+        string id,
```

Comment:
These should ideally start with capital letters by convention. This applies to all the models